.github/workflows/tpl-download-charts.yaml

```diff
@@ -0,0 +1,51 @@
+name: "Download Charts"
+
+on:
+  workflow_call:
+    inputs:
+      registry-source:
+        type: string
+        required: true
+        description: "Repository to pull Helm charts from"
+      target-branch:
+        type: string
+        description: "Branch to commit downloaded Helm charts"
+        default: "charts"
+
+jobs:
+  download-charts:
+    name: "Download charts to local Git repository"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout repository branch"
+        uses: actions/checkout@v2
+        with:
+          ref: "${{ inputs.target-branch }}"
+          fetch-depth: 0
+
+      - name: "Install Helm mirror"
+        run: |
+          cd /tmp
+          wget https://github.com/konstructio/helm-mirror/releases/download/v0.5.0/helm-mirror_linux_x86_64.tar.gz
```

Comment:
One of the inputs imho should be the version of `helm-mirror` now that you're productionizing this so much 😆

Reply:
hahaha done! Maybe we should add alerts (Slack message) for when the job fails.

Reply:
😅 Whoever was the last person to execute the action will get an email notification when they fail.